cmd/erasure-multipart.go

```diff
@@ -1227,12 +1226,6 @@ func (er erasureObjects) CompleteMultipartUpload(ctx context.Context, bucket str
 	ctx = lkctx.Context()
 	defer lk.Unlock(lkctx)
 
-	// Write final `xl.meta` at uploadID location
```

Comment:
So this is already done by renamedata? Nice to get rid of this!

Reply:
yes @klauspost